### PR TITLE
feat: Export stats type to allow for easier usage as a library

### LINKED ISSUE #17: Making unexported types like `careerStats` as exported
For example I need some logic with different modes:
```golang
var stats ovrstat.CareerStats
switch mode {
case "competitive":
  stats = user.Profile.CompetitiveStats
case "quickplay":
  stats = user.Profile.QuickPlayStats
}
```
But now I have to use reflect package and digging type.

### PR DESCRIPTION
**Pull request by jangraefen**
This will export some internal types, so that the usage as a library gets easier.

Resolves #17

### CODE REVIEW

**Comment by jangraefen** [comment]
Hello @s32x,

any chance you take a look? With kind regards 🙂

**Comment by s32x** [comment]
Hey! Sorry for taking so long to look at this. The last few months have been hectic for me. Of course - I'll merge this now.

**Comment by jangraefen** [comment]
Thanks @s32x and no worries 😉

File: ovrstat/models.go [hunk 1]
```diff
@@ -24,14 +24,14 @@ type Rating struct {
...
 }
 
-// statsCollection holds a collection of stats for a particular player
-type statsCollection struct {
-	TopHeroes   map[string]*topHeroStats `json:"topHeroes"`
-	CareerStats map[string]*careerStats  `json:"careerStats"`
+// StatsCollection holds a collection of stats for a particular player
+type StatsCollection struct {
+	TopHeroes   map[string]*TopHeroStats `json:"topHeroes"`
+	CareerStats map[string]*CareerStats  `json:"CareerStats"`
```
Comment (1RV34) [inline ovrstat/models.go:30]: This change seems to make this JSON property stick out, was this intentional?

**Pull request merged**